app/services/database.py

```diff
@@ -435,7 +438,6 @@ def generate_token(self, session,client_id, user_name):
             session.commit()
             self.logger.info(f"Record inserted successfully. ID: {record_model.Id}")
             return record_model
-            print("Generated token:", record_model.Id)
```

Comment:
Ok, also remove extra print from rest of the code base.

Reply:
code Modified and removed the unwanted print statement reflected on this changeset e1ae681